css/css-animations/parsing/animation-computed.html

```diff
@@ -15,22 +15,23 @@
 // <single-animation> = <time> || <easing-function> || <time> ||
 // <single-animation-iteration-count> || <single-animation-direction> ||
 // <single-animation-fill-mode> || <single-animation-play-state> ||
-// [ none | <keyframes-name> ]
-test_computed_value("animation", "1s", "1s ease 0s 1 normal none running none");
-test_computed_value("animation", "cubic-bezier(0, -2, 1, 3)", "0s cubic-bezier(0, -2, 1, 3) 0s 1 normal none running none");
-test_computed_value("animation", "1s -3s", "1s ease -3s 1 normal none running none");
-test_computed_value("animation", "4", "0s ease 0s 4 normal none running none");
-test_computed_value("animation", "reverse", "0s ease 0s 1 reverse none running none");
-test_computed_value("animation", "both", "0s ease 0s 1 normal both running none");
-test_computed_value("animation", "paused", "0s ease 0s 1 normal none paused none");
-test_computed_value("animation", "none", "0s ease 0s 1 normal none running none");
-test_computed_value("animation", "anim", "0s ease 0s 1 normal none running anim");
+// <single-animation-composition> || [ none | <keyframes-name> ]
+test_computed_value("animation", "1s", "1s ease 0s 1 normal none running replace none");
```

Comment:
Is this the usual way we update WPTs for new longhand components of existing shorthands? This change will make the test (as of right now) fail in all engines. Not sure if we want to go to the extent of something like:

```js
let composition = CSS.supports("animation-composition", "replace") ? " replace": "";
test_computed_value("animation", "1s", `1s ease 0s 1 normal none running${composition} none`);
...
```